<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>t2</class>
 <widget class="QWidget" name="t2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="processArrayButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>ArrayButton</string>
   </property>
  </widget>
  <widget class="QComboBox" name="columnTypeComboBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="arrayLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="processMatrixButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>MatrixButton</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="matrixTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="outputPlainTextEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>129</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
